docs/user_guide/user_guide.md

```diff
@@ -39,8 +39,10 @@ A few of the included alerts and reports are dependent on other reports being en
     * Palo Alto Firewall - Network Compromise - Inbound Traffic from Blocked IPs
     * Palo Alto Firewall - Network Compromise - Outbound Traffic to Blocked IPs
         * Dynamically Update Blocked IPs with HoneyDB
-    * Ransomware - Spike in File Writes 
+    * Ransomware - Spike in File Writes
         * Ransomware - Calculate UpperBound for Spike in File Writes
+    * Network Compromise - DDoS Behavior Detected
```

Comment:
We need to remove this from this branch.

Reply:
Done